### DISCUSSION TITLE
How can special pricing arrangements be included in the schema

### DISCUSSION THREAD

**Discussion by kavkuk** [CMSgov/price-transparency-guide#201] [General]
We have some edge cases where special pricing arrangement is made (e.g. employer group level pricing).  Any thoughts around how we can include these cases in the given schema?

**Comment by jltang-kp** [comment]
So to clarify in this situation all other employer groups in that are associated with the same benefit plan would have one rate for services but some employer groups have different pricing for the same services?

As of now I think you might need to duplicate the entire object for services that have special pricing at the employer group level and leverage the description field to notate that those specific services are listed again because of a specific employer group pricing arrangement.

**Comment by shaselton-usds** [comment]
@kavkuk 

Can you provide a little more information with your "employer group level pricing"?  Is it similar to this discussion https://github.com/CMSgov/price-transparency-guide/discussions/175?

**Reply by kavkuk** [reply]
@shaselton-usds No it is not the same as #175.  This would be like for example, pricing based on Age Categories or a Modifier that can change the price of that service/item.  Instead of leveraging the description field to notate these special pricing, can we add a custom field to the In-Network Object?
